wkdehdlr/CoinHelper/app/src/main/java/kr/schoolsharing/coinhelper/tasks/UpbitFragment.kt

```diff
@@ -32,7 +32,7 @@ class UpbitFragment : BaseFragment<FragmentCoinrecyclerBinding>(R.layout.fragmen
             val fragment = UpbitFragment()
             val bundle = Bundle()
 
-            bundle.putString("MARKET_NAME", marketName)
+            bundle.putString(Name.MARKET_NAME.toString(), marketName)
```

Comment:
toString 대신 name 을 사용하시면 됩니다.
```suggestion
            bundle.putString(Name.MARKET_NAME.name, marketName)
```
enum 클래스 이름이 너무 General 합니다.
좀 더 명확하게 명명하셨으면 좋겠습니다.
예를 들면,
```
enum class Market {
    NAME
}
```

Reply:
[090f7db](https://github.com/StudyFork/GoogryAndroidArchitectureStudy/pull/210/commits/090f7db45a2e3740c7972026e9f4048d92eaea6a) 수정했습니다